src/notelistmodel.cpp

```diff
@@ -453,17 +445,17 @@ bool NoteListModel::dropMimeData(const QMimeData *mime, Qt::DropAction action, i
                 auto lastMod = index.data(NoteDeletionDateTime).toDateTime();
                 for (destinationChild = 0; destinationChild < m_noteList.size();
                      ++destinationChild) {
-                    const auto &note = m_noteList[destinationChild];
-                    if (note.deletionDateTime() <= lastMod) {
+                    const auto &refNote = m_noteList[destinationChild];
```

Comment:
Agreed. I'll apply your first suggestion, thanks.

In part because even after I Googled 'inline variables', I'm still a bit confused about how to use them (and they seem to require a C++17 compiler, but I think we still support C++14 ones?)...

But most importantly, it's not cool to make you my personal Wikipedia every time... 😆

Reply:
Erm, I don't mean asking the compiler to inline the variable, but to just do it by changing the code to:
```c++
if (m_noteList[destinationChild].deletionDateTime() <= lastMod) {
```

Reply:
LMAO... Okay, I gotcha now.